conf/app.routes

```diff
@@ -346,9 +346,24 @@ POST        /which-years-change-claim                            controllers.aut
 GET         /change-which-years-change-claim                     controllers.authenticated.ChangeWhichTaxYearsController.onPageLoad(mode: Mode = CheckMode)
 POST        /change-which-years-change-claim                     controllers.authenticated.ChangeWhichTaxYearsController.onSubmit(mode: Mode = CheckMode)
 
-GET        /confirmation                       controllers.ConfirmationController.onPageLoad()
-
-GET        /fifth-Industry-Options                               controllers.FifthIndustryOptionsController.onPageLoad(mode: Mode = NormalMode)
-POST       /fifth-Industry-Options                               controllers.FifthIndustryOptionsController.onSubmit(mode: Mode = NormalMode)
-GET        /change-FifthIndustry-Options                         controllers.FifthIndustryOptionsController.onPageLoad(mode: Mode = CheckMode)
-POST       /change-FifthIndustry-Options                         controllers.FifthIndustryOptionsController.onSubmit(mode: Mode = CheckMode)
+GET        /confirmation                                         controllers.ConfirmationController.onPageLoad()
+
+GET        /fifth-industry-options                               controllers.FifthIndustryOptionsController.onPageLoad(mode: Mode = NormalMode)
+POST       /fifth-industry-options                               controllers.FifthIndustryOptionsController.onSubmit(mode: Mode = NormalMode)
+GET        /change-fifthIndustry-options                         controllers.FifthIndustryOptionsController.onPageLoad(mode: Mode = CheckMode)
```

Comment:
/change-fifthIndustry-options should be /change-fifth-industry-options, keep in mind that this is what gets displayed in the URL